CHANGELOG.md

```diff
@@ -1,15 +1,30 @@
 # Changelog
 
-## v0.34.3 
+## v0.34.4
+
+*February 11, 2021*
+
```

Comment:
Should we say that this release fixes a memory leak and all clients are recommended to upgrade?

Reply:
Mmm yes, I like this! Will update (in another PR unfortunately).